packages/lesswrong/components/comments/CommentsItem/CommentsItem.jsx

```diff
@@ -141,7 +141,7 @@ class CommentsItem extends Component {
   }
 
   render() {
-    const { comment, currentUser, frontPage, nestingLevel=1, showPostTitle, classes, post } = this.props
+    const { comment, currentUser, postPage, nestingLevel=1, showPostTitle, classes, post } = this.props
```

Comment:
Is `postPage` a boolean, or some sort of collection / database object? Perhaps we should move towards requiring propTypes to help with this sort of thing.